ci/kokoro/install/Dockerfile.centos-7

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2019 Google LLC
```

Comment:
I guess this is because of the auto generation, and tedious to keep the original year, but strictly saying we should keep the original year for the copyright notice, at least our layers won't be happy.